File: citiesfetcher/src/main/java/Source.java

/**
 * Created by metin on 03.04.2016.
 */
public enum Source {
    Diyanet("Diyanet.gov.tr"),
    Fazilet("FaziletTakvimi.com"),
    IGMG("IGMG.org"),
    Semerkand("SemerkandTakvimi.com"),
    NVC("NamazVakti.com"),
    Morocco("habous.gov.ma"),
    Malaysia("e-solat.gov.my");

    public String text;

    Source(String text) {
        this.text = text;
    }

}

File: app/src/main/java/com/metinkale/prayer/times/times/Source.kt
package com.metinkale.prayer.times.times

import com.metinkale.prayer.R


enum class Source(val fullName: String, val drawableId: Int? = null) {
    Diyanet("Diyanet", R.drawable.ic_times_ditib),
    IGMG("IGMG", R.drawable.ic_times_igmg),
    London("LondonPrayerTimes.com"),
    NVC("NamazVakti.com", R.drawable.ic_times_namazvakticom),
    Semerkand("SemerkandTakvimi", R.drawable.ic_times_semerkand),
    Calc("Calc"),
    CSV("CSV");
}
